README.md

```diff
@@ -0,0 +1,36 @@
+# FairGrounds
+News Rating App
+
+<h2>Database setup</h2>
+
+We are using a Postgres database that have three tables set up as follow:
+
+<h3>Users:</h3>
+    <p class="indent">username character varying(200) UNIQUE NOT NULL,<br>
+    password character varying(200) UNIQUE NOT NULL,<br>
+    userid serial PRIMARY KEY,<br>
+    leaning integer,<br>
+    email character varying(200) UNIQUE NOT NULL</p>
+<h3>Articles:</h3>
+    <p>articleid serial PRIMARY KEY,<br>
+    topic character varying(200) NOT NULL,<br>
```

Comment:
In looking at the information that is being pulled in from the API, topic is not a referenced object. "Title" is the keyword that is used. We can keep "topic" but change it from "not null".

Reply:
This is what the sql file is (copied directly).  Could you change the SQL file first and then I (or you) can change the documentation to match?  I did a function yesterday that the server doesn't need to change when the database is changed, so the SQL file just needs to be formed how you need it.

Reply:
I'm happy to make that change.